cpu/sam0_common/include/periph_cpu_common.h

```diff
@@ -342,40 +342,51 @@ void gpio_init_mux(gpio_t pin, gpio_mux_t mux);
  */
 static inline int sercom_id(const void *sercom)
 {
+    int ret = SERCOM_INST_NUM;
```

Comment:
I think you don't need this.

Reply:
You are right. I also added the `assert(false);` before returning. Not that there's an increase of code size by 48 bytes with this PR.

Reply:
I would assume the increase in size is only with `DEVHELP = 1`